### DISCUSSION TITLE
V1 feeds endpoint error

### DISCUSSION THREAD

**Discussion by nhinzevt** [transitland/transitland#340] [Q&A] [answered]
The following API call returns the error message below:

https://transit.land/api/v1/feeds?bbox=-122.4183,37.7758,-122.4120,37.7858&api_key=xxx

[NoMethodError] undefined method `onestop_id' for nil:NilClass"

I have tested other coordinates and these fail also.

**Comment by drewda** [comment, marked as answer]
Thanks for your patience. We have now had a chance to address this issue (https://github.com/transitland/transitland-datastore/issues/1314) and it is fixed in production.
